src/api/errors/levels/FatalError.ts

```diff
@@ -0,0 +1,20 @@
+import BaseError from './BaseError'
+
+class FatalError extends BaseError {
+  cause?: Error
+
+  constructor (title: string, message: string, cause?: Error) {
```

Comment:
Parentheses must be attached to the constructor.
```suggestion
  constructor(title: string, message: string, cause?: Error) {
```